src/utils/userAgent.ts

```diff
@@ -7,6 +7,8 @@ const { name } = parser.getBrowser()
 export const isMobile = type === 'mobile' || type === 'tablet'
 const platform = parser.getOS().name
 export const isIOS = platform === 'iOS'
+export const isAndroid = platform === 'Android'
```

Comment:
wondering if we should include Chrome/Chromium OS in this - https://uniswapteam.slack.com/archives/C02GYG8TU12/p1698949931008619